<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OscilloScope</class>
 <widget class="QWidget" name="OscilloScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bottom</string>
  </property>
  <widget class="QLineEdit" name="offset_POSLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>600</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;位置曲线偏移量（°）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="srComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>600</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;速度曲线分度值&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>0.5rpm/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1rpm/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2rpm/div</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="offset_CURLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>620</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电流曲线偏移量（mA）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scopeEnablePushButton">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>600</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;运行/暂停示波器&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>II</string>
   </property>
  </widget>
  <widget class="QComboBox" name="xDivComboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>620</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;速度曲线分度值&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>1s/div</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="offset_SPDLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>600</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;速度曲线偏移量（rpm）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tgCURPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;目标电流曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>tgCUR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tgSPDPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;目标速度曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>tgSPD</string>
   </property>
  </widget>
  <widget class="QComboBox" name="crComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>620</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电流曲线分度值&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>20mA/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60mA/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100mA/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200mA/div</string>
    </property>
   </item>
  </widget>
  <widget class="QwtPlot" name="plotPOS">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>399</y>
     <width>685</width>
     <height>195</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tgPOSPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;目标位置曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>tgPOS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="prComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>600</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;位置曲线分度值&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>10°/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50°/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100°/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000°/div</string>
    </property>
   </item>
  </widget>
  <widget class="QwtPlot" name="plotSPD">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>202</y>
     <width>685</width>
     <height>195</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rlCURPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;实际电流曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>rlCUR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rlSPDPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>620</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;实际速度曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>rlSPD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rlPOSPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;实际位置曲线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>rlPOS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ScanFrequencyComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>620</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;采样周期选择&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>10ms/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50ms/div</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100ms/div</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="adjustGroupComboBox">
   <property name="geometry">
    <rect>
     <x>1107</x>
     <y>220</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;切换PID调整组显示：1-低载PID参数，2-中载PID参数，3-重载PID参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QwtPlot" name="plot">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>685</width>
     <height>195</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
